src/utils/docs.jl

```diff
@@ -111,9 +111,19 @@ function open_doc()
     end
 end
 
+function start_doc_preview_server(;open_browser::Bool = true, port::Int = 8000)
+  build_dir = normpath(Oscar.oscardir, "docs", "build")
+  cmd = "using Pkg; Pkg.activate(temp = true);
+         Pkg.add(\"LiveServer\"); using LiveServer;
+         LiveServer.serve(dir = \"$build_dir\", launch_browser = $open_browser, port = $port);"
+  live_server_process = run(`julia -e $cmd`, wait = false)
+  atexit(_ -> kill(live_server_process))
```

Comment:
Did you verify this works? Because when I just tried this by copy&pasting this code, `kill` was just ignored. I suspect this was because I am using `juliaup`, which actually forks `julia` into a subprocess...

Reply:
Okay... I'm very sure it worked for me at some point. I will try again.

Reply:
Could you try again? Maybe it is fixed by https://github.com/oscar-system/Oscar.jl/pull/2574/commits/7af4c60c74ff770d47f81569206a08a13e23723e as @benlorenz suggested.

Reply:
I tried it now with `juliaup` and it works (with https://github.com/oscar-system/Oscar.jl/commit/7af4c60c74ff770d47f81569206a08a13e23723e). Without that commit it doesn't, so I assume that this is fixed.